seed/views/v3/organizations.py

```diff
@@ -1324,21 +1329,25 @@ def report_export(self, request, pk=None):
         count_sheet.write(data_row_start, data_col_start + 1, 'Properties with Data', bold)
         count_sheet.write(data_row_start, data_col_start + 2, 'Total Properties', bold)
 
-        base_sheet.write(data_row_start, data_col_start, 'ID', bold)
-        base_sheet.write(data_row_start, data_col_start + 1, request.query_params.get('x_label'), bold)
-        base_sheet.write(data_row_start, data_col_start + 2, request.query_params.get('y_label'), bold)
-        base_sheet.write(data_row_start, data_col_start + 3, 'Year Ending', bold)
-
         agg_sheet.write(data_row_start, data_col_start, request.query_params.get('x_label'), bold)
         agg_sheet.write(data_row_start, data_col_start + 1, request.query_params.get('y_label'), bold)
         agg_sheet.write(data_row_start, data_col_start + 2, 'Year Ending', bold)
 
         # Gather base data
         cycles = self.get_cycles(params['start'], params['end'], pk)
+        matching_columns = Column.objects.filter(organization_id=pk, is_matching_criteria=True, table_name="PropertyState")
```

Comment:
Should this be checking whether to query for table_name "PropertyState" or "TaxlotState"? PropertyState has matching column `pm_property_id` where TaxLotState has `jurisdiction_tax_lot_id`. 

If this feature is exclusive to properties then ignore this comment